packages/foodie-backend/src/app.ts

```diff
@@ -17,19 +18,21 @@ app.use(helmet());
 app.use(cors());
 app.set('trust proxy', 1); // trust first proxy
 
-app.use('/api/v1', [AuthRouter, UserRouter]);
+app.use('/api/v1', [AuthRouter, UserRouter, VendorRouter]);
 
 /**
  * TODO
  * - Create 404 error
  */
 // catch 404 and forward to error handler
 app.use((req: Request, res: Response, next: any) => {
```

Comment:
I will suggest custom error handler

Reply:
I have started working on custom errors for the app, would add it in now